crates/kornia-io/src/jpeg.rs

```diff
@@ -69,10 +70,30 @@ pub fn read_image_jpeg_rgb8(file_path: impl AsRef<Path>) -> Result<Image<u8, 3>,
 ///
 /// A grayscale image with a single channel _(mono8)_.
 pub fn read_image_jpeg_mono8(file_path: impl AsRef<Path>) -> Result<Image<u8, 1>, IoError> {
-    read_image_jpeg_internal(file_path)
+    read_image_jpeg_impl(file_path)
 }
 
-fn read_image_jpeg_internal<const N: usize>(
+/// Decodes a JPEG image with three channel (rgb8) from Raw Bytes.
+///
+/// # Arguments
+///
+/// - `image` - A mutable reference to your `Image`
+/// - `bytes` - Raw bytes of the jpeg file
+pub fn decode_image_jpeg_rgb8(image: &mut Image<u8, 3>, bytes: &[u8]) -> Result<(), IoError> {
```

Comment:
```suggestion
pub fn decode_image_jpeg_rgb8(src: &[u8], dst: &mut Image<u8, 3>) -> Result<(), IoError> {
```
to keep the same convention order like this, first immutable followed by mutables